iree/compiler/Dialect/HAL/Target/LLVM/LLVMTargetOptions.h

```diff
@@ -39,6 +39,11 @@ struct LLVMTargetOptions {
   // and benchmarking
   bool debugSymbols = true;
 
+  // Define and declare kinds of Sanitizer as enum
+  // the order in enum should be same as one in flat buffer schema
+  enum SanitizerKind { None = 0, Address };
```

Comment:
Let's declare the enum outside the struct, e.g., https://github.com/google/iree/blob/87c96f26bca9e14196071235eeff2c3ce575677c/iree/compiler/Dialect/VM/Conversion/TargetOptions.h#L25-L29

Here is cpp files, so we should follow the naming style, e.g., `kNone`, kAddress`, etc.

And please use `enum class`. See https://abseil.io/tips/86